pkg/apis/core/types.go

```diff
@@ -2031,6 +2031,8 @@ type ResourceRequirements struct {
 	// otherwise to an implementation-defined value
 	// +optional
 	Requests ResourceList
+	// Integrate with OpenStack flavorRef
```

Comment:
ResourceRequirements is a generic data structure for both VMs and containers. But flavor is specific to VM. Should we add flavorRef into VMSpec?

Reply:
In our previous discussion on OpenStack & Kubernetes Types Mapping:
![image](https://user-images.githubusercontent.com/19650434/88716606-2758da00-d0d4-11ea-9b17-00182afc92f4.png)

It shows that flavorRef is mapping to Resources. That's why I put it here.

Reply:
I incline to agree with Xiaoning here. flavor of VMs is more like a label level ( or other characteristic in the spec ) for a VM.

Reply:
Thank you for your suggestion, Yunwen.

@XiaoningDing Do you mean PodSpec when you said VMSpec?